installing/installing_ibm_power/installation-config-parameters-ibm-power.adoc

```diff
@@ -0,0 +1,12 @@
+:_content-type: ASSEMBLY
+[id="installation-config-parameters-ibm-power"]
+= Installation configuration parameters for {ibmpowerProductName}
+include::_attributes/common-attributes.adoc[]
+:context: installation-config-parameters-ibm-power
+:platform: IBM Power
+
+toc::[]
+
+Before you deploy an {product-title} cluster, you provide a customized `install-config.yaml` installation configuration file that describes the details for your environment.
```

Comment:
```suggestion
Before you deploy an {product-title} cluster, provide a customized `install-config.yaml` installation configuration file that describes the details for your environment.
```

Reply:
I think you can omit this 2nd "you".

Reply:
For consistency with the other providers I would prefer keeping the "you". WDYT @mjpytlak ?

Reply:
> For consistency with the other providers I would prefer keeping the "you". WDYT

Yes @SNiemann15 please keep the second "you".